From bbc95ad5036dd1f2caa90a2bf1e829ca2c485783 Mon Sep 17 00:00:00 2001
From: Ran Wang <ran.wang_1@nxp.com>
Date: Wed, 22 Nov 2017 16:12:51 +0800
Subject: [PATCH 197/211] powerpc/T104xRDB: Fix endian access issue on EHCI intinalization

This issue is exposed after commit 9000eddbae0d
("drivers/usb/ehci: Use platform-specific accessors"),
the wrong endian way of EHCI controller programing will cause
USB function down.

Signed-off-by: Ran Wang <ran.wang_1@nxp.com>
---
 include/configs/T104xRDB.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/configs/T104xRDB.h b/include/configs/T104xRDB.h
index 0035e67..f1b127f 100644
--- a/include/configs/T104xRDB.h
+++ b/include/configs/T104xRDB.h
@@ -640,6 +640,7 @@ $(SRCTREE)/board/freescale/t104xrdb/t1042d4_sd_rcw.cfg
 #ifdef CONFIG_USB_EHCI_HCD
 #define CONFIG_USB_EHCI_FSL
 #define CONFIG_EHCI_HCD_INIT_AFTER_RESET
+#define CONFIG_EHCI_DESC_BIG_ENDIAN
 #endif
 #endif
 
-- 
1.7.1

